tools/slang-unit-test/unit-test-secure-crt.cpp

```
@@ -0,0 +1,617 @@
+    // sizeOfBuffer == 0
+    {
+        char buf[4];
+        errno = 0;
+        SLANG_CHECK(sprintf_s(buf, 0, "%s", "x") == -1);
+        SLANG_CHECK(errno == EINVAL);
+    }
+    // Overflow — output exceeds buffer
+    {
+        char buf[4];
+        int n = sprintf_s(buf, sizeof(buf), "%s", "toolong");
+        SLANG_CHECK(n == -1);
```

Cppcheck (2.20.0)

[style] The function 'BFloat16ToFloat' is never used.

(unusedFunction)